sw-manifest.js

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
I feel like using rollup-plugin-workbox-inject may be a better approach but it's possible I don't understand the situation. Let me see if I can articulate the pros:

- We could remove this file and pass the config in our rollup.config.js. I see this as a huge win.
- We would switch to using the `self.__WB_MANIFEST` approach for injection which inlines the manifest (I think the importScripts approach will incur a bit of network overhead on the initial install). The upside to this is that it's potentially a tad faster, and well documented in the workbox docs.

Cons:
- rollup-plugin-workbox-inject (and other workbox inject tools) use find/replace to match self.__WB_MANIFEST which is a bit unsafe.

Reply:
> To counter your first point, if we just had a naïve 'rollup.config.js' with the rollup-plugin-workbox-inject plugin, we'd be forced to run all of Rollup early but all the resources wouldn't be available yet either (since Eleventy requires the client Rollup build to finish).

I'm not sure I'm following this point. In the npm script don't we already do a full rollup build before we tell eleventy to start watching?

Reply:
That's entirely the point. At that stage in the build process we don't yet know all the assets we need to include in the SW manifest.